lib/browser/api/browser-window.js

```diff
@@ -144,6 +144,12 @@ BrowserWindow.fromWebContents = (webContents) => {
   }
 }
 
+BrowserWindow.fromBrowserView = (browserView) => {
```

Comment:
Should add a `return null` at the bottom of this (super nit picky but otherwise the return type is technically `undefined`)

Reply:
Solid idea, done :white_checkmark: